deploy/osps/default/osp-rhel.yaml

```diff
@@ -418,6 +418,9 @@ spec:
             systemctl restart systemd-modules-load.service
             sysctl --system
 
+            {{- /* restart sshd.service */}}
```

Comment:
Can you please add a reason here as well like why do we need to restart `sshd.service`?

Reply:
> Can you please add a reason here as well like why do we need to restart `sshd.service`?

The reason is kinda empyrical: in Azure the ssh server becomes unresponsive and it is not possible to ssh into the node after the second cloud-init run.

Reply:
I mean add the reason as a comment in the file 😁